faq.md

```diff
@@ -234,55 +301,112 @@ several layouts.
 
 ## Will user guides be included?
 
-Perifractic designed a nostalgic traditional spiral bound Getting Started guide with an awesome (top secret) cover co-created by him and Trevor Storey. The manual includes a BASIC programming guide. The rest of the team are currently finishing up the finer details of the manual. Further docs are already available in the Downloads section of this website. The working title for the spiral bound guide is “Just the BASICs: Getting started with the Commander X16”. There will also be “Assembling Assembly” and a programmer’s reference guide. PDFs will likely be an option.
+Perifractic designed a nostalgic traditional spiral bound Getting Started guide
+with an awesome (top secret) cover co-created by him and Trevor Storey. The
+manual includes a BASIC programming guide. The rest of the team are currently
+finishing up the finer details of the manual. Further docs are already available
+in the Downloads section of this website. The working title for the spiral bound
+guide is “Just the BASICs: Getting started with the Commander X16”. There will
+also be “Assembling Assembly” and a programmer’s reference guide. PDFs will
+likely be an option.
 
 ## Why not use the Parallax Propeller chip, FPGA, Arduino, Teensy, Pi Pico, etc?
 
-We will not be using the Propeller for these reasons:
-Most of its capabilities are on par with or inferior to our custom FPGA.
-There are issues when putting it on the system bus related to CPU read attempt speed call and answer
-The Propeller costs the same as if not more than the faster FPGA
+We will not be using the Propeller for these reasons: Most of its capabilities
+are on par with or inferior to our custom FPGA. There are issues when putting it
+on the system bus related to CPU read attempt speed call and answer The
+Propeller costs the same as if not more than the faster FPGA
+
+## Seriously, if the X16 doesn't have X, I don't want it.
+
+Everyone has their own definition of their dream computer. This is David's.
+
+Engineering is ultimately an exercise in compromise. To meet the goals of the
+project at a cost target, not everything and the kitchen sink could be included.
+Likewise, the X16 represents a stable platform with which to write amazing
+software. If it had too many features (like a bunch of FPGA accelerators) it
+would be hard to use and compatibility, testability, etc. would have been far
+more difficult.
+
+Building a community around the X16 was just important as the hardware, and that
+means keeping the base system static and simple, so developers have a "fixed
+target" to work from and so that both users and students have something
+consistent to work with.
+
+The X16 is a spiritual successor to the Commodore PET, and we feel that it fills
```

Comment:
I haven't heard it called a PET successor before, only to either the C64 or Commodore's 8-bit computers as a whole.

Reply:
It's in part 2 of David's original video (https://youtu.be/sg-6Cjzzg8s?list=PLfABUWdDse7bKGFshxR0itdHBhjUj86SX&t=290). The X16 architecture heavily resembles the PET in its use of off the shell logic chips. I think it's an important distinction since the C64 uses CPLDs and while folks compare it to the C64 in terms of performance, making the distinction here that it's not actually a C64 sequel is a good point to make as I think it adds clarify and might help avoid confusion as to why the X16 works the way it does in some respects.

Reply:
Not sure if there's any significance to the timestamp you linked, as he's going on about the VIC-20 there and only mentions the PET once in the entire video (in relation to using the VERA with Commodore systems). Off-the-shelf/readily-available parts, I remember being a key goal of this project, though, yeah.

All this made me completely forget I was going to comment on the prior paragraph. "Building a community around the X16 was just important as the hardware" - it still is important. Nitpicking on tenses? Maybe.

Reply:
I think we could say the "Commodore 8-Bit computers", since the X16 iterates on all their features. 

80 columns like the PET... but higher resolution. 
More colors and sound than the 64.
Faster than the 128. 

The only thing we don't have yet is CP/M mode... but with the expansion slot, that can happen too.

Reply:
Ah yes that should be present tense good catch. And I also like the "Commodore 8-bit computers" bit. That encompasses it nicely. I'll update the PR shortly.

Reply:
PR updated. I also noticed the FAQ said "While many folks might have their own definition of their dream computer, this is David's." twice so I removed it from this section, tried to make it a bit more direct and kept with the present tense.